tensorflow_probability/examples/BUILD

```diff
@@ -91,10 +52,11 @@ py_library(
 py_test(
     name = "bayesian_neural_network_test",
     size = "small",
-    srcs = ["bayesian_neural_network.py"],
+    srcs = [bayesian_neural_network.py"],
```

Comment:
Looks like this line dropped a quote - would you mind adding it back in? That'll help me pull this in to our internal repo.

Reply:
Apologies, completely missed this. Added this back in.